ansible/roles/ansible-tvault-contego-extension/tasks/object_store_service.yml

```diff
@@ -5,52 +5,25 @@
 
 - debug: msg="external s3:{{TVAULT_CONTEGO_EXT_S3}}" verbosity={{verbosity_level}}
 
-- name: Install tvault-object-store layer on centos/RedHat if Python2
-  package:
-    name:
-      - python-s3fuse-plugin-cent7
-    state: latest
-  when: >
-      ((ansible_distribution_major_version=="7" and ansible_distribution | lower =="centos" and PYTHON_VERSION=="python2") or
-      (ansible_distribution_major_version=="7" and ansible_distribution | lower =="redhat" and PYTHON_VERSION=="python2") or
-      (ansible_distribution_major_version=="8" and ansible_distribution | lower =="centos" and PYTHON_VERSION=="python2") or
-      (ansible_distribution_major_version=="8" and ansible_distribution | lower =="redhat" and PYTHON_VERSION=="python2"))
-
 - name: Install packages on compute node | centos-release-openstack-{{OPENSTACK_DIST}}
```

Comment:
we can remove this task, its already done in install_rhel task

Reply:
removed